extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java

```diff
@@ -105,23 +98,23 @@ public class OidcCommonConfig {
     public static class Credentials {
 
         /**
-         * Client secret which is used for a `client_secret_basic` authentication method.
-         * Note that a 'client-secret.value' can be used instead but both properties are mutually exclusive.
+         * Optional. The client secret used by the `client_secret_basic` authentication method.
```

Comment:
Suggest to remove `Optional.` and instead, after `<p/>` clarify that it "must be set unless a secret is set in {@link #clientSecret} or {@link #jwt} client authentication is required"

Reply:
Updated. Thanks.